src/helpers/api.js

```diff
@@ -3,7 +3,8 @@ import { getDomain } from 'helpers/getDomain';
 
 export const api = axios.create({
   baseURL: getDomain(),
-  headers: { 'Content-Type': 'application/json' }
+  headers: { 'Content-Type': 'application/json',
+  "authorization": (localStorage.getItem("token")) ? localStorage.getItem("token"): null}
```

Comment:
Can you remove the configuration (authorization header) in the other files where it is used?

Reply:
Will do

Reply:
Done, Removed from 1 File (Header) couldnt find anywhere else